src/v3/src/transformer/selectAuthenticator/utils.ts

```diff
@@ -36,6 +36,42 @@ const getAuthenticatorDataSeVal = (authenticatorKey: string, methodType?: string
   return '';
 };
 
+const reorderAuthenticatorButtons = (
+  authButtons: AuthenticatorButtonElement[],
+  options: IdxOption[],
+): AuthenticatorButtonElement[] => {
+  if (authButtons.length <= 1) {
+    return authButtons;
+  }
+  const ovRemediation = options.find((option) => option.relatesTo?.key === AUTHENTICATOR_KEY.OV);
+  const methodType = (ovRemediation?.value as Input[])?.find(({ name }) => name === 'methodType');
+  if (!methodType?.options?.find((option: IdxOption) => option.value === 'signed_nonce')) {
+    return authButtons;
+  }
+  const fastpassAuthenticator = authButtons.find(
+    (button) => button.options.actionParams?.['authenticator.methodType'] === 'signed_nonce',
+  );
+  if (!fastpassAuthenticator) {
+    return authButtons;
+  }
+
+  const updatedAuthenticatorBtns = authButtons.filter(
+    (button) => button.options.actionParams?.['authenticator.methodType'] !== 'signed_nonce',
+  );
+
+  // Re-arrange fastpass in options based on deviceKnown
+  // If deviceKnown is set, set fastpass as the first option in the list
+  // otherwise, place it as the last item in the list of OV options
```

Comment:
Ah interesting so there is actually some client-side ordering that is expected? Is this logic from v2?

Reply:
Oh yes! 

https://github.com/okta/okta-signin-widget/blob/master/src/v2/ion/ui-schema/ion-object-handler.js#L61